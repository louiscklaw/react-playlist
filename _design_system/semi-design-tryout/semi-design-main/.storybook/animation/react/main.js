const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
