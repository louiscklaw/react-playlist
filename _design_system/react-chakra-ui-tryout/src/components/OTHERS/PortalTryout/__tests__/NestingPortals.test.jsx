import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { ChakraProvider } from '@chakra-ui/react';

import { NestingPortals } from '../';

describe('test NestingPortals tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(
      <ChakraProvider>
      <NestingPortals />
      </ChakraProvider>
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
