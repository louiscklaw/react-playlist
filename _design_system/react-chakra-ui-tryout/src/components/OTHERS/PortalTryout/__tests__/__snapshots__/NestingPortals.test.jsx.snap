// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test NestingPortals tryout helloworld 1`] = `
Array [
  <div>
    <div
      className="css-8v5jrc"
    />
  </div>,
  <span />,
]
`;
